LL-master patched from ED-master

EPSIL0N opened the pull request:
I believe the it was a minor issue and I appear to have been able to resolve it but I would rather merge into master and do testing with the download and play method rather than immediately merge into the live testing branch that pull request #13 is suggesting.

pickten committed:
Merge pull request #73 from eternaldensity/master

update

pickten committed:
Add retro multibuy, nerf photo

Oh joy new ridiculously long line!

pickten committed:
Fix retro multibuy

pickten committed:
Update boosts.js

pickten committed:
remove splosion nerf, slight buff

pickten committed:
Update boosts.js

pickten committed:
Update boosts.js

pickten committed:
Uber-nerfed Fragment because it was too powerful

Now ocean blue/fragment don't let you go too high. I'll probably need to re-buff Fragment. We'll see.

pickten committed:
Clean up RetroAct a bit, remove some barely-possible stuff

pickten committed:
Update castle.js

pickten committed:
Fix initial reactions

pickten committed:
possible fix to Argy

Might completely fail.

pickten committed:
Possible fix to LL's no-draw issue

pickten committed:
Fix erroneous "wait for it"

pickten committed:
More "scientific" stats views

pickten committed:
Merge pull request #162 from eternaldensity/master

Update to master

pickten committed:
Minor fix

Probably nothing serious, does break negpix monuments, we'll figure that out later

Commit:
Merge pull request #1371 from pickten/variegation

Add retro multibuy, nerf photo

pickten committed:
Fix some retroact typos

That functions a bit of a mess, sorry.

Commit:
Merge pull request #1373 from pickten/variegation

Fix some retroact typos

pickten committed:
Update dragonoverview.js

charmsRace committed:
Some cleanup

charmsRace committed:
Cleanup kitkat and PG for TaT

charmsRace committed:
NWWI TaT

charmsRace committed:
LA, Vamp refactor, test Retroreflector

charmsRace committed:
Merge remote-tracking branch 'upstream/master'

pickten committed:
Fix bad TaT-ONG()

pickten committed:
also fix time-travel to 0

pickten committed:
Fix unkept story

Commit:
Merge pull request #1374 from Calamitizer/master

TaT cleanup

Commit:
Merge pull request #1375 from pickten/overview

Might not fix overview

charmsRace committed:
Panopticon testing & badge

pickten committed:
Fix a minor issue

pickten committed:
Probable fix for -2->-1 TTT

pickten committed:
Possible fix for missing wait for it

charmsRace committed:
mayo

Commit:
Merge pull request #1377 from pickten/overview

Minor bugfixes

pickten committed:
Standartize approx

pickten committed:
Update options.js

pickten committed:
Work to a notif silencer

pickten committed:
Keep log working with the notif silencer

Commit:
[AM] Add cloudsync via gists

Commit:
Merge pull request #1378 from erp-lsf/master

[AM] Add cloudsync via gists

charmsRace committed:
Abattoir fix maybe?

charmsRace committed:
Aperture unlock fix

Commit:
Merge pull request #1379 from Calamitizer/master

Some small fixes

pickten committed:
Possible fix?

Commit:
Merge branch 'overview' of https://github.com/pickten/Sandcastle-Builder into pickten-overview
Changed option NotifSilence to notifsilence for consistency
Conflicts:
	options.js

Commit:
Merge branch 'pickten-overview'

EPSIL0N committed:
Merge branch 'ED-master' into LL-master'

Merge Conflicts:
	badges.js
new entries conflict for 'Two Pots O\' Gold' and 'One in a Million'
added 'One in a Million' from incoming ED to the bottom of the list

All other changes appear to have autoresolved.

EPSIL0N commented on boosts.js:
```diff
@@ -5286,7 +5286,7 @@ Molpy.DefineBoosts = function() {
 		if(times && Molpy.IsEnabled('Mario')) {
 			var l = Molpy.Boosts['Mario'].bought;
-			var runs = [10,20,100,500,1000][Molpy.options.approx];
+			var runs = 25;
```
Why did we replace the runs options with a fixed 25?

Re: pickten. I thought the issue had been suppressed but I'll stop fussing over this seeing as the clean up is already done and I haven't studied how Photoelectricity or pretty much anything requiring alternate timelines to unlocks works.

pickten replied:
Because if the number was too big, it could crash computers, so 25 was safer. In general, I removed the option entirely. Besides, I used 25 in Photoelectricity and it didn't make sense to have different ones.

EPSIL0N commented on boosts.js:
```diff
@@ -12716,10 +12732,10 @@ Molpy.Coallate = function(){
-			} else if (this.bought == 10) {
+			} else if (this.bought == this.limit) {
 				this.countdown = 0;
 				this.power = 0;
```
The Author of the boost has said it is intentional so I revoke my concerns.

EPSIL0N committed:
Update castle.js

Changed an old reference to Aperture .times which was changed to the standardized .limit elsewhere.

EPSIL0N committed:
Abbatoir owned conditions reflect limit

Changed some conditionals so that Exit through the Abbatoir doesn't act 'owned' until the limit is bought.

EPSIL0N committed:
Hopefully made 'Wisdom of the Ancients' less useless

EPSIL0N commented:
I don't think anything is broken by these changes but I wasn't playing most of today so I don't know if we should move them to the gh-pages as well right away.
